operations/configuration.md

```diff
@@ -139,13 +139,39 @@ Returns all countries supported by the API.
             "Code": "US",
             "EnglishName": "United States of America"
         }
+    ],
+    "CountrySubdivisions": [
+        {
+            "Code": "AU-NSW",
+            "CountryCode": "AU",
+            "EnglishName": "New South Wales"
+        },
+        {
+            "Code": "AU-QLD",
+            "CountryCode": "AU",
+            "EnglishName": "Queensland"
+        }
+    ],
+    "CountryAlliances": [
+        {
+            "Code": "SCHENGEN",
+            "EnglishName": "Schengen Area",
+            "CountryCodes":[
+                "AT",
+                "BE",
+                "CZ",
+                ...
```

Comment:
Is it okay that we don't specify the full list?

Reply:
We don't have full list of countries in documentation.  IMHO it is, but wait for Honza.